rlog/state-separation.mdx

```diff
@@ -0,0 +1,1255 @@
+---
+title: 'Nescience - A user-centric state separation architecture'
+date: 2024-08-07 12:00:00
+authors: moudy
+published: false
+slug: Nescience-A-state-separation-architecture
```

Comment:
Replace:
"Nescience-A-state-separation-architecture"

with:
"Nescience-state-separation-architecture"

(it's just a cleaner URL)